test/stresstest/stress.py

```diff
@@ -0,0 +1,47 @@
+########################################################################
+# Copyright (c) 2020 Robert Bosch GmbH
```

Comment:
If we add new files, should not copyright year be current year? I.e. 2022?

Reply:
Of course :)